src/components/pages/DatasetsList.jsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import ReactRouterPropTypes from 'utils/react-router-prop-types';
+import PropTypes from 'prop-types';
+import { Eye } from 'react-feather';
+import SearchBox from 'components/form-components/SearchBox';
+
+const DatasetPagination = (props) => {
+  const {
+    first,
+    last,
+    total,
+  } = props;
+  return (
+    <div className="text-gray-600">
+      Показано: з {first} по {last} з {total} елементів
+    </div>
+  );
+};
+
+DatasetPagination.propTypes = {
+  first: PropTypes.string.isRequired,
+  last: PropTypes.string.isRequired,
+  total: PropTypes.string.isRequired,
+};
```

Comment:
first, last, total can be PropTypes.number

Reply:
++